python/llm/src/bigdl/llm/transformers/models/utils.py

```diff
@@ -153,11 +153,10 @@ def apply_rotary_pos_emb(q, k, cos, sin, position_ids, model_family):
         k_embed = (k * cos) + (rotate_half(k) * sin)
         return q_embed, k_embed
     elif model_family == "gptj":
-        cos = torch.repeat_interleave(cos[:, :, None, :], 2, 3)
-        sin = torch.repeat_interleave(sin[:, :, None, :], 2, 3)
-        q_embed = (q * cos) + (rotate_every_two(q) * sin)
-        k_embed = (k * cos) + (rotate_every_two(k) * sin)
-        return q_embed, k_embed
+        torch.ops.torch_ipex.apply_rotary_embedding_two_qk(
```

Comment:
Don't change this. This `apply_rotary_pos_emb` is used for both cpu and xpu.

Reply:
Divided the cpu logic.